haproxy/dockerdir/usr/bin/add_pxc_nodes.sh

```diff
@@ -64,7 +64,7 @@ cat <<-EOF > "$path_to_haproxy_cfg/haproxy.cfg"
       external-check command /usr/local/bin/check_pxc.sh
 EOF
 
-    echo "${#NODE_LIST[@]}" > $path_to_haproxy_cfg/AVAILABLE_NODES
+    echo "${#NODE_LIST_REPL[@]}" > $path_to_haproxy_cfg/AVAILABLE_NODES
```

Comment:
why?

Reply:
not matter for us which var is used but ${#NODE_LIST[@]} has a wrong value due to change of structure 
https://github.com/percona/percona-docker/blob/3418f6c2ab64e2b954bcb4e8050ecab6d6aa7033/haproxy/dockerdir/usr/bin/add_pxc_nodes.sh#L43